tensorhive/api/api_specification.yml

```diff
@@ -83,3 +117,38 @@ definitions:
       created_at:
         type: string
         format: date-time
+        
+  CreateReservationEventController:
+    type: object
+    properties:
+      title: { type: string }
+      description: { type: string }
+      nodeId: { type: integer}
+      userId: { type: integer}
+      start:
+        type: string
+        format: date-time
+      end:
+        type: string
+        format: date-time
+      # TODO Add user id, node id
+  ReservationEvent:
+    type: object
+    properties:
+      id:
+        type: integer
+        format: int64
+      title: { type: string }
+      description: { type: string }
+      nodeId: { type: integer}
```

Comment:
nodeId suggest a whole node, the reservation granularity should be per device.
Rename to deviceId? What if the reservation concerns more devices, do we want to create multiple events or one event with multiple devices in it?

Reply:
We definitely must discuss this kind of dilemmas.